readinglist/views/article.py

```diff
@@ -56,6 +57,7 @@ class ArticleSchema(ResourceSchema):
     class Options:
         readonly_fields = ('url', 'stored_on') + \
             ResourceSchema.Options.readonly_fields
+        unique_fields = ('url', 'resolved_url')
```

Comment:
I made my best to explain this point the last commit.

``resolved_url`` should be unique as well as ``url``.

In the first version, URL resolution is performed by client. In the next version, URLs will be resolved synchronously during creation, in order to make sure ``resolved_url`` is maintained unique.

(*I remember a conversation about this with the product owner, but we can re-discuss it*)